draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1531,11 +1534,10 @@ are:</t>
 <t><list style="hanging">
 
 <t hangText="Running:">The check list is neither Completed yet nor
-Failed yet.  Check lists are initially set
-to the Running state.
+Failed yet. Check lists are initially set to the Running state.
 </t>
 
-<t hangText="Completed:">The check list has a selected candidate pair
+<t hangText="Completed:">The check list contains a selected pair
```

Comment:
"nominated" may make more sense in this context. I think of it as "once you have a nominated pair for each component, the state changes to completed, and the pairs become selected". There's a paragraph below that basically states this.

Reply:
Stupid question: if "completed check list" and "selected pairs" means the same thing, why do we need both? :)